d/vibed/dub.json

```
@@ -1,8 +1,8 @@
 {
   "name": "server",
   "dependencies": {
-    "vibe-d:http": "~>0.10.0",
-    "vibe-d:tls": "*"
+      "vibe-http": "~>1.1.0",
+      "vibe-stream:tls": "~>1.1.0"
```

I think this is enough, no @cyrusmsk 

```suggestion
      "vibe-d": "~>0.10.0"
```

---

No, because by default it is using tls. So to switch off the TLS you need to specify it explicitly. vibe-d has vibe-http in dependency.